docs/wiki/End-user-Guide/installation/install-server.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Server
+---
+
+# Server
+
+## How to Install Cleanroom Loader on a Server
+
+### Steps
+
+1. Delete the `libraries` folder inside your server directory.
+2. ⚠️ **Warning:** Ensure your server modlist does not include problematic or deprecated mods.  
+   See [Preparing your modpack](/wiki/End-user-Guide/preparing-your-modpack) for more details.
+3. Download the latest `cleanroom-x.x.x-installer.jar` from the [Releases page](https://github.com/CleanroomMC/Cleanroom/releases) and run it.
+4. In the installer GUI, select **Install server** and choose your server’s directory.
+5. Add [Fugue](https://www.curseforge.com/minecraft/mc-mods/fugue) to your modlist  
+   *(it will not load in a Forge environment)*
+6. Add [Scalar Legacy](https://www.curseforge.com/minecraft/mc-mods/scalar-legacy) to your modlist  
+   *(it will not load in a Forge environment)*
+7. *(Optional)* Add JVM arguments to your server’s launch file as described in the [JVM Arguments](/wiki/End-user-Guide/args) section.
+
+---
```

Comment:
as before, the header used already creates a page break, so this creates a second